camera/rollup.config.js

```diff
@@ -4,7 +4,7 @@ export default {
     {
       file: 'dist/plugin.js',
       format: 'iife',
-      name: 'capacitorCamera',
+      name: 'WatchdutyCamera',
```

Comment:
Tiniest of nits: the diff changes from lower-camel to upper-camel case. Is that intentional?

Reply:
I'm not sure since it's been like this, but I agree. Updated.